dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerTest.java

```diff
@@ -1120,4 +1120,24 @@ public void testFindLinkageProblems_grpcAndGuava() throws IOException {
                 "has class binary name"))
         .contains("io.grpc.internal.DnsNameResolver");
   }
+
+  @Test
+  public void testFindLinkageProblems_referenceToNonexistentMethodForJava8() throws IOException {
+    // Protobuf-java 3.12.4 contained wrong byte code to reference non-existent method in JRE 1.8
```

Comment:
protobuf-java 3.12.4 references a Java 11 method that does not exist in Java 8

Reply:
Updated.